sphinx_rtd_theme/footer.html

```diff
@@ -2,10 +2,10 @@
   {% if (theme_prev_next_buttons_location == 'bottom' or theme_prev_next_buttons_location == 'both') and (next or prev) %}
     <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
       {% if next %}
-        <a href="{{ next.link|e }}" class="btn btn-neutral float-right" title="{{ next.title|striptags|e }}" accesskey="n">{{ _('Next') }} <span class="fa fa-arrow-circle-right"></span></a>
+        <a href="{{ next.link|e }}" class="btn btn-neutral float-right" title="{{ next.title|striptags|e }}" accesskey="n" rel="next" rev="prev">{{ _('Next') }} <span class="fa fa-arrow-circle-right"></span></a>
```

Comment:
I checked it in the W3C Validator and found that apparently the rev attribute is obsolete. Though it is in the W3C HTML 5.1 Recommendation and there is no warning about that there.

> The rev attribute on the a element is obsolete. Use the rel attribute instead, with a term having the opposite meaning.

OK, then I'll remove it.